application/src/main/java/org/togetherjava/tjbot/commands/free/FreeHelpChannelsRoutine.java

```diff
@@ -0,0 +1,65 @@
+package org.togetherjava.tjbot.commands.free;
+
+import net.dv8tion.jda.api.JDA;
+import net.dv8tion.jda.api.entities.TextChannel;
+import net.dv8tion.jda.api.utils.TimeUtil;
+import org.jetbrains.annotations.NotNull;
+import org.togetherjava.tjbot.commands.Routine;
+
+import java.time.OffsetDateTime;
+import java.util.concurrent.TimeUnit;
+
+/**
+ * Routine that automatically marks inactive help channels as free.
+ * <p>
+ * The routine runs every {@value CHECK_INTERVAL_MINUTES} minutes, and checks for help channels that
+ * have been inactive for more than {@value INACTIVE_TIME_MINUTES} minutes.
+ * </p>
+ */
+public class FreeHelpChannelsRoutine implements Routine {
```

Comment:
Bad class name. It should be called according to its purpose. So sth like `FreeInactiveHelpChannelsRoutine`. I.e. the emphasis is on the fact that it operates on inactive channels.